WalletWasabi/Blockchain/Keys/KeyManager.cs

```diff
@@ -582,38 +611,35 @@ public ExtKey GetMasterExtKey(string password)
 		/// Make sure there's always clean keys generated and indexed.
 		/// Call SetMinGapLimit() to set how many keys should be asserted.
 		/// </summary>
-		public bool AssertCleanKeysIndexed(bool? isInternal = null)
+		public IEnumerable<HdPubKey> AssertCleanKeysIndexed(bool? isInternal = null)
 		{
-			var generated = false;
+			var newNewKeys = new List<HdPubKey>();
```

Comment:
why `newNewKeys`? maybe you meant `newKeys`

Reply:
yes